tools/releasing/deploy_staging_jars.sh

```diff
@@ -40,6 +40,11 @@ cd ..
 
 echo "Deploying to repository.apache.org"
 
+COMMON_OPTIONS="-Prelease,docs-and-source -DskipTests -DretryFailedDeploymentCount=10"
+
 echo "Deploying Scala 2.11 version"
-$MVN clean deploy -Prelease,docs-and-source -DskipTests -DretryFailedDeploymentCount=10
+$MVN clean deploy $COMMON_OPTIONS -Pscala-2.11
```

Comment:
this doesn't work as you cannot specify the `-P` flag multiple times. One of the `-P` arguments will be completely ignored (can't remember which)

This is a perfect example why property-based activations are so useful.

Reply:
I think using multiple `-P` flags works, I checked it with `mvn help:active-profiles -Pscala-2.12 -Pscala-11` (which is a stupid example but works).

Nevertheless, I'll change to profile activation so that we can have a default profile. However, this will only work well as long as we only have two scala versions because I can use `!scala-2.12` and `scala-2.12`.